newIDE/app/src/stories/componentStories/Profile/Subscription/CancelReasonDialog.stories.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import muiDecorator from '../../../ThemeDecorator';
+import paperDecorator from '../../../PaperDecorator';
+import AuthenticatedUserContext, {
+  type AuthenticatedUser,
+} from '../../../../Profile/AuthenticatedUserContext';
+import {
+  fakeSilverAuthenticatedUser,
+  fakeNotAuthenticatedUser,
+} from '../../../../fixtures/GDevelopServicesTestData';
+import CancelReasonDialog from '../../../../Profile/Subscription/CancelReasonDialog';
+import AlertProvider from '../../../../UI/Alert/AlertProvider';
+
+export default {
+  title: 'Subscription/CancelReasonDialog',
+  component: CancelReasonDialog,
+  decorators: [paperDecorator, muiDecorator],
+};
+
+const CancelReasonDialogWrapper = ({
+  authenticatedUser,
+}: {
+  authenticatedUser: AuthenticatedUser,
+}) => {
+  return (
+    <AlertProvider>
+      <AuthenticatedUserContext.Provider value={authenticatedUser}>
+        <CancelReasonDialog
+          onClose={() => action('on close')()}
+          onCloseAfterSuccess={action('on close after success')}
+        />
+      </AuthenticatedUserContext.Provider>
+    </AlertProvider>
+  );
+};
+
+export const Loading = () => (
+  <CancelReasonDialogWrapper authenticatedUser={fakeNotAuthenticatedUser} />
```

Comment:
Almost no reason to use a wrapper I think if we just limit the providers to AuthenticatedUserContext (the wrapper is nice, but it makes thing less flexible/force you to duplicate the types)

Reply:
yeah I copied from another story, will update